gradle.properties

```diff
@@ -1 +1,3 @@
 org.gradle.caching=true
+org.gradle.jvmargs=-Xmx4g -Dfile.encoding=UTF-8
```

Comment:
Lets make this maybe just `-Xmx2g` (I know it's only an example, but 4 is quite high for a plain Java project)

Reply:
Done.